packages/oruga/vitest.workspace.ts

```diff
@@ -0,0 +1,28 @@
+import { defineWorkspace } from "vitest/config";
+
+const browserTestPattern = "src/**/*.browser.test.{ts,js}";
+
+export default defineWorkspace([
```

Comment:
Can't this be integrated in the current existing vite.config file?

Reply:
As it turns out, it can be integrated.